DEBIAN/changelog

```diff
@@ -0,0 +1,5 @@
+zcash (1.0.0) UNRELEASED; urgency=medium
+
+  * Initial packaging for Debian.
+
+ -- Zcash Company <team@z.cash>  Tue, 27 Sep 2016 12:05:17 -0700
```

Comment:
@nathan-at-least What should we standardize upon for the name of our team, and our e-mail address? This is also something we'll need to decide upon for the attributes of the master signing key.